src/app/app.config.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+angular.module('linagora.esn.linshare')
+  .config(injectApplicationMenu);
+
+  function injectApplicationMenu(dynamicDirectiveServiceProvider) {
+    let linshare = new dynamicDirectiveServiceProvider.DynamicDirective(true, 'linshare-application-menu', { priority: -10 });
+
+    dynamicDirectiveServiceProvider.addInjection('esn-application-menu', linshare);
```

Comment:
If I remember well, we don't need this anymore and we removed this in other SPAs, right?